05-bootstrap-prep.md

```diff
@@ -0,0 +1,88 @@
+# Prep for cluster bootstrapping
+
+Now that the [hub-spoke network is provisioned](./04-networking.md), the next step in the [AKS secure Baseline reference implementation](./) is preparing what your AKS cluster should be bootstrapped with.
+
+## Expected results
+
+Container registries often have a lifecycle that extends beyond the scope of a single cluster. They can be scoped broadly at organizational or business unit levels, or can be scoped at workload levels, but usually are not directly tied to the lifecycle of any specific cluster instance. For example, you may do blue/green _cluster instance_ deployments, both using the same container registry. Even though clusters came and went, the registry stays in tact.
```

Comment:
is "In tact" ok here ?  Or the adjective "intact" is better usage ?  I'm not sure, but just in case

Reply:
Good catch.